tensorflow/core/platform/default/build_config/BUILD

```diff
@@ -91,20 +91,20 @@ filegroup(
 cc_library(
     name = "cuda",
     data = [
-        "//third_party/gpus/cuda:{}".format(cuda_library_path("cudart")),
+        "@local_config_cuda//cuda:{}".format(cuda_library_path("cudart")),
     ],
     linkopts = select({
-        "//third_party/gpus/cuda:darwin": [
-            "-Wl,-rpath,third_party/gpus/cuda/lib",
-            "-Wl,-rpath,third_party/gpus/cuda/extras/CUPTI/lib",
+        "@local_config_cuda//cuda:darwin": [
+            "-Wl,-rpath,external/local_config_cuda/cuda/lib",
+            "-Wl,-rpath,external/local_config_cuda/cuda/extras/CUPTI/lib",
```

Comment:
Does that work? With the deepexecroot change it will break, can you try ../local_config_cuda instead?